apps/web/pages/apps/installed.tsx

```diff
@@ -233,10 +230,10 @@ export default function IntegrationsPage() {
           success={({ data }) => {
             return data.items.length > 0 ? (
               <>
-                <IntegrationsContainer variant="conferencing" />
-                <CalendarListContainer />
-                <IntegrationsContainer variant="payment" className="mt-8" />
-                <IntegrationsContainer variant="other" className="mt-8" />
+                <IntegrationsContainer items={data.items} variant="conferencing" />
+                <CalendarListContainer items={data.items} />
+                <IntegrationsContainer items={data.items} variant="payment" className="mt-8" />
+                <IntegrationsContainer items={data.items} variant="other" className="mt-8" />
```

Comment:
We're discarding the benefits of using tRPC here, we should lookup how to pass initial data to queries instead.

Reply:
Also by passing these props we're losing the portability of these "containers"

Reply:
Reverted